test/mailers/previews/admin_mailer_preview.rb

```diff
@@ -13,4 +13,9 @@ def new_conversation
     notification = Notification.where(type: NewConversationNotification.to_s).first
     AdminMailer.with(record: notification, recipient: User.first).new_conversation
   end
+
+  def potential_hire
+    notification = Notification.where(type: PotentialHireNotification.to_s).first
```

Comment:
Is one of these created when running the database seeds? If not, can you update an existing record to make sure one of these notifications is created?